caravel/assets/javascripts/SqlLab/main.css

```diff
@@ -29,11 +32,22 @@
     padding-bottom: 5px;
 }
 
-.scrollbar {
-    position: relative;
-    width: 100%;
-    overflow-y: auto;
-    height: 100%;
+.scrollbar-container {
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  height: 95%;
+}
+
+.scrollbar-content {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  right: 0px;
+  bottom: 0px;
+  overflow: scroll;
+  margin-right: 0px;
+  margin-bottom: 100px;
```

Comment:
I think it's just the margin between panel/left-bar and bottom of browser window, in case they are too close and look like being cutoff